app/javascript/v3/App.vue

```diff
@@ -15,6 +15,7 @@ export default {
   mounted() {
     this.setColorTheme();
     this.listenToThemeChanges();
+    this.setLocale(window.chatwootConfig.selectedLocale);
```

Comment:
What happens if the `selectedLocale` is `undefined`? Do we have to worry about that? Also should we be worried about `chatwootConfig` being `undefined`

Reply:
It is never undefined. We take it from I18n.locale. default is en.